modules/client-configuration/pages/registration-bootstrap.adoc

```diff
@@ -35,7 +35,10 @@ Bootstrap scripts based on Python 2 must be re-created for openSUSE Leap 15 an
 If you register openSUSE Leap 15 or SLE 15 systems using Python 2, the bootstrap script fails.
 ====
 
-
+[IMPORTANT]
+====
+After migrating from an older version of SUSE Manager to a newer version, we strongly recommend re-generating the existing bootstrap scripts before onboarding new systems to prevent any potential issues.
+====
```

Comment:
```suggestion

ifeval::[{suma-content} == true]

[IMPORTANT]
====
After migrating from an older version of {productname} to a newer version, we strongly recommend re-generating the bootstrap scripts before onboarding new systems to prevent any potential issues.
====

endif::[]
```